Epoch 0:   0%|                                                                                                                                                                                 | 0/104 [00:00<?, ?it/s]Traceback (most recent call last):
  File "/home/sehwan/MIIL/CAL-Refactoring/dlt/main.py", line 181, in <module>
    main()
  File "/home/sehwan/MIIL/CAL-Refactoring/dlt/main.py", line 54, in main
    TrainLoopCAL(accelerator=accelerator, model=model, diffusion=noise_scheduler,
  File "/home/sehwan/MIIL/CAL-Refactoring/dlt/trainers/cal_trainer.py", line 129, in train
    self.CAL_train(epoch, self.diffusion_mode)
  File "/home/sehwan/MIIL/CAL-Refactoring/dlt/trainers/cal_trainer.py", line 185, in CAL_train
    model_predict = self.model(uncond_batch, noisy_batch, t) # -> [64,max_comp, 518] : xy(2) + wh(2) + r(1) + z(1) + image_feature(512)
  File "/home/sehwan/anaconda3/envs/CAL/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/sehwan/anaconda3/envs/CAL/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/sehwan/MIIL/CAL-Refactoring/dlt/models/CAL.py", line 107, in forward
    additional_column = torch.zeros(key_padding_mask.shape[0], 1, dtype=torch.bool).cuda()
  File "/home/sehwan/anaconda3/envs/CAL/lib/python3.9/site-packages/torch/cuda/__init__.py", line 302, in _lazy_init
    torch._C._cuda_init()
RuntimeError: Unexpected error from cudaGetDeviceCount(). Did you run some cuda functions before calling NumCudaDevices() that might have already set an error? Error 803: system has unsupported display driver / cuda driver combination